Arjona/template/simple_functions.py

```diff
@@ -10,3 +10,26 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+		
+# Python program to check if  
```

Comment:
It would be more useful if this was a function, rather than a standalone program.